### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by voarsh2** [openark/orchestrator#1469]
Keeping it simple - seems this project is dead? Not maintained anymore?

**Comment by pbnsh** [comment]
yup, see http://code.openark.org/blog/mysql/reducing-my-oss-involvement-and-how-it-affects-orchestrator-gh-ost

**Comment by fzyzcjy** [comment]
Sad to hear that :(

**Comment by voarsh2** [comment]
Ah yes... and no one has thought "let's get together" and form a reputable fork? :D
